documentation/developers/documentation.md

```diff
@@ -0,0 +1,26 @@
+# Documentation
+
+Sphinx est utilisé pour la génération de la documentation depuis des pages écrites en Markdown (via le [parser MyST](https://myst-parser.readthedocs.io/en/latest/)).
+
+## Génération du site web de documentation
+
+Pour la génération:
+
+```bash
+# install aditionnal dependencies
+python -m pip install -U -r requirements/documentation.txt
```

Comment:
this line doesn't work locally but it works within the python:bullseye container.

Reply:
what is your version of python ?

Reply:
an old one ^^ 2.7.x. Because it works inside docker (python 3.11.x), it's ok for me. ;) I will add or modify a Dockerfile to build the doc locally (maybe... if I have time^^).